### PR TITLE
Send both notification and requests counts

### PR DESCRIPTION
**Pull request by AdityaKhatri**

### CODE REVIEW

**Review by annshress — APPROVED** [review]

**Review by annshress — APPROVED** [review]

**Comment by tnagorra** [comment]
@AdityaKhatri naaani? nandakure? assignee @tnagorra ?

File: apps/notification/tests/test_apis.py [hunk 1]
```diff
@@ -106,6 +109,36 @@ def create_join_request(self, project, user=None):
...
+        )
+
+    # TODO: To test notification after creation of entry comments
+    def create_entry(self, project, **fields):
+        lead = self.create(Lead, project=project)
+        return self.create(
+            Entry, lead=lead, project=lead.project,
+            analysis_framework=lead.project.analysis_framework,
+            **fields
+        )
```
Comment (thenav56) [inline apps/notification/tests/test_apis.py]: This already exists in the Base TestCase.
https://github.com/the-deep/server/blob/3bebe249667406624a6316e77de3bf4dadfc2832/deep/tests/test_case.py#L258-L265
https://github.com/the-deep/server/blob/3bebe249667406624a6316e77de3bf4dadfc2832/deep/tests/test_case.py#L226-L234

File: apps/notification/tests/test_apis.py [hunk 2]
```diff
@@ -106,6 +109,36 @@ def create_join_request(self, project, user=None):
...
+        """Create entry comment"""
+        user = user or self.create(User)
+        entry = self.create_entry(project)
+        entry_comment = self.create(
+            EntryComment,
+            entry=entry,
+            assignees=[self.user],
+            text='test comment',
+        )
+        return entry_comment
```
Comment (thenav56) [inline apps/notification/tests/test_apis.py]: Defined but not used.
For notification creation, you will need to use **captureOnCommitCallbacks**. Something like this
https://github.com/the-deep/server/blob/3bebe249667406624a6316e77de3bf4dadfc2832/apps/entry/tests/test_entry_comment.py#L205-L206

Comment (AdityaKhatri) [reply apps/notification/tests/test_apis.py]: I've removed this for now

File: apps/notification/tests/test_apis.py (hunk 1 above)
Comment (AdityaKhatri) [reply apps/notification/tests/test_apis.py]: I've removed them

**Review by annshress — APPROVED** [review]
tests added nice.

**Review by thenav56 — APPROVED** [review]

**Pull request merged**